convex/myFunctions.ts

```diff
@@ -168,3 +168,91 @@ export const updateUser = mutation({
     });
   },
 });
+
+//function to submit a new feature request
+export const submitFeatureRequest = mutation({
+  args: {
+    title: v.string(),
+    description: v.string(),
+  },
+  handler: async (ctx, args) => {
+    const userId = await getAuthUserId(ctx);
+
+    if (!userId) {
+      console.log("User not authenticated");
+      return null;
+    }
+
+    await ctx.db.insert("featureRequest", {
+      userId,
+      title: args.title,
+      description: args.description,
+      status: "open",
+    });
+  },
+});
+
+//function to get user stats
+export const getUserStats = query ({
+  args: {
+    userId: v.id("users"),
+  },
+  handler: async (ctx, { userId }) => {
+    const user = await ctx.db.get(userId);
+
+    if (!user) {
+      console.log("User not authenticated");
```

Comment:
use logger.

Reply:
convex dosent come with a log method should I install an external logger  ?

Reply:
See logger.ts